frontend/src/layout/navigation-3000/components/Breadcrumbs.tsx

```diff
@@ -36,7 +36,6 @@ export function Breadcrumbs(): JSX.Element | null {
         <div
             className="Breadcrumbs3000"
             style={
-                // eslint-disable-line react/forbid-dom-props
```

Comment:
accident?

Reply:
I tried to add it back, but the linter auto-removes this rule - seems it's not applicable here as there's no eslint error when removed